itinerant-iterators/app/obfuscate/find_bounds.py

```diff
@@ -0,0 +1,39 @@
+import requests
+
+
+def find_bounds(image_file: str, text: str, regex: bool = False) -> list[tuple[int, int, int, int]]:
+    """Gets bounding boxes of text from an image based on text to match, which can be regex.
+
+    Args:
+        image_file (str): File path to image
+        text (str): Text to match, or regex
+        regex (bool, optional): If the text is regex. Defaults to False.
+
+    Returns:
+        list[tuple[int, int, int, int]]: List of bounding boxes for each character to hide
+    """
+    data = {
+        "text": text,
+        "regex": regex
+    }
+    files = {
+        'file': open(image_file, 'rb')
+    }
+    bounds = requests.post("https://itinerantiterators.hipeople21.repl.co", data=data, files=files)
+    return bounds.json()
```

Comment:
Any sort of networking can easily fail for any number of reasons. The response status code should be checked before returning the data. Additionally, there should be a try... except... where this function is called that catches `requests.HTTPError` and emits a clear error to the user.

```suggestion
    response = requests.post("https://itinerantiterators.hipeople21.repl.co", data=data, files=files)
    response.raise_for_status()
    return response.json()
```